.gitignore

```diff
@@ -100,4 +100,4 @@ Scripts/
 .mise.toml
 
 # Support for uv.lock will come in a future PR. See #4796
-uv.lock
+uv.lock
```

Comment:
very nitpicky: if you add a trailing newline, the file will notbe changed by this PR at all 👼